### DISCUSSION TITLE
JSX react modules in the dapp?

### DISCUSSION THREAD

**Discussion by mpixelz** [hashlips-lab/nft-erc721-collection#458] [Q&A]
Hi, im trying to add menus and other sections in the dapp.. all the templates and repos i have found for reference work with either JS or JSX files..
whenever i try to create a component in jsx file and try to import in the dapp, it doesn't work.. can anyone please point me in the direction what i need to do to be able to use jsx components in this dapp :( please.

**Comment by liarco** [comment]
Hi @mpixelz,
At the moment I actively maintain my projects by fixing bugs that people report here on GitHub. For any support for custom implementations/features or previews of future updates I only provide this kind of support to holders of ODC tokens (https://opendevs.io). This helps me focusing my resources on people who support my work and it allows me to work on new tools to bring out in the future with less distractions.

If you are already a holder you can follow the instructions on the website to join our private Discord channel and ask your questions there. We already have some examples and content that might help, but we can also help with more specific stuff.

I hope this helps and I wish you a wonderful day! ☀️
